integration-tests/test/mrf_lp.spec.ts

```diff
@@ -1707,9 +1706,606 @@ describe('Liquidity Pool Test', async () => {
 
       const postxBOBAAmount = await xBOBAToken.balanceOf(env.l2Wallet.address)
 
-      expect(prexBOBAAmount).to.deep.eq(
-        postxBOBAAmount.add(exitAmount)
+      expect(prexBOBAAmount).to.deep.eq(postxBOBAAmount.add(exitAmount))
+    })
+  })
+
+  describe('Onramp batch tests', async () => {
```

Comment:
for all the tests could add check to verify the exact amount is deducted from the balance on L1

Reply:
Added